src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -265,7 +265,6 @@ void BaseCouplingScheme::extrapolateData(DataMap &data)
       PRECICE_DEBUG("Extrapolate data: " << pair.first);
       PRECICE_ASSERT(pair.second->oldValues.cols() > 1);
       Eigen::VectorXd &values       = pair.second->values();
-      pair.second->oldValues.col(0) = values;  // = x^t
       values *= 2.0;                           // = 2*x^t
       values -= pair.second->oldValues.col(1); // = 2*x^t - x^(t-1)
       utils::shiftSetFirst(pair.second->oldValues, values);
```

Comment:
In principle you also store data in `shiftSetFirst`, right? Question is whether you want to change this in this PR or later?

Reply:
I would prefer changing this later. https://github.com/BenjaminRueth/precice/tree/package_time overhauls this part of the extrapolation. I think changing it in this PR is not really necessary.